### PR TITLE
IGVF-1005-max-read-length

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

**Comment by ian-whaling** [comment]
mean_read_length type was changed from integer to number since that's the output type from the checkfiles script.

File: src/igvfd/schemas/changelogs/sequence_file.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ## Changelog for *`sequence_file.json`*
 
+### Schema version 3
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/sequence_file.md:3]: Minor typo (schema ver 4)

File: src/igvfd/upgrade/file.py [hunk 2]
```diff
@@ -28,3 +28,23 @@ def ref_file_3_4(value, system):
...
+
+
+@upgrade_step('sequence_file', '3', '4')
+def sequence_file_3_4(value, system):
+    # https://igvf.atlassian.net/browse/IGVF-1005
+    if 'minimum_read_length' in value and value['minimum_read_length'] > 300000000:
+        value['minimum_read_length'] = 300000000
+        notes = value.get('notes', '')
+        notes += f' The minimum read length previously exceeded the upper limit of 300000000 so was set to 300000000, please assign the appropriate length.'
+        value['notes'] = notes
```
Comment (jenjou) [inline src/igvfd/upgrade/file.py:40]: Can this result in a extra space at the front of the string?

File: src/igvfd/schemas/changelogs/sequence_file.md (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/sequence_file.md]: Changed.

File: src/igvfd/upgrade/file.py (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/upgrade/file.py:43]: Yes it can, changed.

**Review by jenjou — APPROVED** [review]

**Pull request merged**